packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -95,6 +99,11 @@ export class StsTokenManager {
     return null;
   }
 
+  // TODO(lisajian): Fill in once refresh listener is added
+  private async getTokenAndTriggerCallback(): Promise<string | null> {
```

Comment:
This should have the right signature (and then ignore unused warnings).

Reply:
SG, added method signature back and noted for future!